src/providers/DevicesProvider/AudioOutputProvider.tsx

```diff
@@ -59,11 +59,20 @@ const AudioOutputProvider: React.FC = ({ children }) => {
       }
     }
 
+    const callback = (deviceLabels: DeviceLabels): void => {
+      if (deviceLabels === DeviceLabels.Audio || deviceLabels === DeviceLabels.AudioAndVideo) {
+        initAudioOutput();
+      }
+    };
```

Comment:
Yeah, you are right. Since we the device permission request only depends on the `deviceLabelTrigger` now. We could simply use `init*()` as callback here. Addressed.